src/manifests/kubeadm/control-plane.yaml

```diff
@@ -11,6 +11,8 @@ localAPIEndpoint:
   bindPort: 6443
 nodeRegistration:
   taints:
+    - key: node-role.kubernetes.io/control-plane
+      effect: NoSchedule
```

Comment:
Won't the control plane be underutilised with this new taint?

Reply:
Actually kubeadm adds by default this taint, but because we added cilium taints in the kubeadm config this vanished. I think we should separate control-plane node from the other nodes for the educational purpose.